packages/core/src/common/externals/NodeJsExternals.ts

```diff
@@ -169,12 +152,13 @@ class ChokidarWatcherWrapper extends EventEmitter implements FsWatcher {
 
 	constructor(watcher: chokidar.FSWatcher) {
 		super()
-		this.#watcher = watcher
-			.on('ready', () => this.emit('ready'))
-			.on('add', (path) => this.emit('add', uriFromPath(path)))
-			.on('change', (path) => this.emit('change', uriFromPath(path)))
-			.on('unlink', (path) => this.emit('unlink', uriFromPath(path)))
-			.on('error', (e) => this.emit('error', e))
+		this.#watcher = watcher.on('ready', () => this.emit('ready')).on(
+			'add',
+			(path) => this.emit('add', uriFromPath(path)),
+		).on('change', (path) => this.emit('change', uriFromPath(path))).on(
+			'unlink',
+			(path) => this.emit('unlink', uriFromPath(path)),
+		).on('error', (e) => this.emit('error', e))
```

Comment:
`preferSingleLine: true` definitely ruined this lol but in general I'm fine with these changes

Reply:
_I can change it to 99 lmao_

Reply:
I fixed the formatting of this file manually.

Reply:
99 feels weird, id be happier with 90 or even 95 as a rounder number. but /shrug